authors/apps/articles/views.py

```diff
@@ -342,3 +342,174 @@ def delete(self, request, slug):
         data = {"article":article_serializer.data}
         data["message"] = "unfavourited"
         return Response(data, status.HTTP_200_OK)
+
```

Comment:
@Ramon3162 Fix the spacing here. Just add an extra line

Reply:
Done

Reply:
Done.